src/atomicassets.cpp

```diff
@@ -466,6 +466,35 @@ ACTION atomicassets::createtempl(
     ).send();
 }
 
+/**
+* Deletes a template if the issued supply is zero
+* @required_auth authorized_editor, who is within the authorized_accounts list of the collection
+**/
+ACTION atomicassets::deltemplate(
+    name authorized_editor,
+    name collection_name,
+    int32_t template_id
+) {
+    require_auth(authorized_editor);
+
+    auto collection_itr = collections.require_find(collection_name.value,
```

Comment:
This check is already done in check_has_collection_auth

Reply:
Correct, ty. check_has_collection_auth will be slightly reworked after all of these merges to implement the CPU optimizations.